### ISSUE TITLE
14.11.0 causes gem size to balloon to 200MB+

### ISSUE THREAD

**Issue by danielmklein** [plaid/plaid-ruby#401]
With the release of version 14.11.0, the size of the installed `plaid` gem is now somewhere around 213 MB. I have confirmed this on our Heroku dynos, on my MacBook Pro workstation, and on the gem's [rubygems.org listing](https://rubygems.org/gems/plaid/versions/14.11.0).

After some deeper digging, it looks like essentially all of the size increase comes from extra `.gem` files in the `pkg` directory. For example, version 14.10.0 only has `plaid-14.7.0.gem` in its `pkg` directory, wherease version 14.11.0 has all of the following:
```
plaid-14.9.0.gem
plaid-14.6.0.gem
plaid-14.3.0.gem
plaid-14.1.0.gem
plaid-14.0.0.gem
plaid-14.0.0.beta.5.gem
plaid-14.0.0.beta.3.gem
plaid-13.2.0.gem
```

Is there any other info I can provide to help out here? This enormous size greatly impacts `bundle install` times as well as making our app intermittently undeployable on Heroku.

# EDIT

So I noticed another thing -- checking rubygems.org, I bumped up gem size with "pushed by" for all 14.x versions, looking for any correlations (table below). There seems to be a correlation where versions pushed by @stephenjayakar tend to be bigger. I'm wondering if there could be something about their environment that's causing the extra stuff to show up in `pkg` in the resulting gem?

Stephen, I apologize if this feels at all finger-pointy -- you seem hella cool, and we appreciate your and everyone else's work on this gem. Just trying to provide extra data to help y'all avoid this problem in the future! ❤️ 

| version | size | pushed by |
| --- | --- | --- |
| 14.11.0 | 205 MB | stephenjayakar
| 14.10.0 | 8.15 MB | mnguyen-plaid
| 14.9.0 | 103 MB |  stephenjayakar
| 14.8.0 | 586 KB | arbitton
| 14.7.0 | 4.08 MB | mnguyen-plaid
| 14.6.0 | 77 MB | stephenjayakar
| 14.5.0 | 18.7 MB | andchen
| 14.4.0 | 14.9 MB | andchen
| 14.3.0 | 12.8 MB | stephenjayakar
| 14.2.0 | 281 KB | cfarmerplaid
| 14.1.0 | 8.55 MB | stephenjayakar
| 14.0.0 | 2.13 MB | stephenjayakar

**Comment by stephenjayakar** [comment]
@danielmklein 
yo thanks for the thorough investigation. i'm going to dig into this rn 

don't sweat it it's probably something wrong with my configuration. it's also good data that it only happens on my machine / not with other releases.

**Comment by stephenjayakar** [comment]
@danielmklein just released 14.11.1 and that should fix the issue for all releases :). thanks so much for the callout. lmk if i can help any more

**Comment by danielmklein** [comment]
Thanks @stephenjayakar! Really appreciate the quick turnaround here!
